bundles/org.openhab.core/src/test/java/org/openhab/core/library/items/NumberItemTest.java

```diff
@@ -209,4 +210,61 @@ public void testKelvinToMired() {
 
         assertThat(item.getState().format("%.0f mired"), is("370 mired"));
     }
+
+    @Test
+    public void stateDescriptionUnitUsedWhenStateDescriptionPresent() {
```

Comment:
Unit tests methods name convention applied (also no public recommended)
```suggestion
    void testStateDescriptionUnitUsedWhenStateDescriptionPresent() {
```